doc/lua_api.md

```diff
@@ -9148,7 +9150,13 @@ gets activated (not loaded!)
     action = function(pos, node, dtime_s) end,
     -- Function triggered for each qualifying node.
     -- `dtime_s` is the in-game time (in seconds) elapsed since the block
-    -- was last active
+    -- was last active.
+
+    bulk_action = function(pos_list, dtime_s) end,
+    -- Function triggered with a list of all applicable node positions at once.
+    -- This can be provided as an alternative to `action` (not both).
+    -- Available since `minetest.features.bulk_lbms` (5.10.0)
+    -- `dtime_s`: see above
```

Comment:
If one lbm adds more nodes, are these nodes included in the remaining lbms of the current block activation?
(Edit: Looks like no.)

I think we should specify such things, or explicitly write down that it's unspecified.

Also:
* Are positions guaranteed to be in one block?
* If in one block, is it guaranteed that it's all applicable nodes in the block?
* Are there order guarantees?
  (Please guarantee none of the above.)
* Btw, for future optimization, a flag in definition to allow false positives might be cool.